console/src/config.rs

```diff
@@ -28,6 +29,11 @@ pub struct Config {
 
     #[clap(flatten)]
     pub(crate) view_options: ViewOptions,
+
+    /// The amount of time resource that have been dropped
+    /// will be displayed in the console.
+    #[clap(long = "retain-for", default_value = "5h", parse(try_from_str = parse_retain_for))]
```

Comment:
i think the default value here basically  means that completed tasks will never be removed (for all practical purposes), which is a significant change from the current behavior. i think in order to get something closer to what the console does on `main`, we should probably set a much smaller default value, like
```suggestion
    #[clap(long = "retain-for", default_value = "6s", parse(try_from_str = parse_retain_for))]
```
(since we previously retained completed tasks for 6 updates, and updates default to coming every second, this should be _about_ the same as what we were doing before).

Reply:
another thought, we may want to have some way to _totally disable_ expiring completed tasks? in most cases you could just set a very long duration, but if i want to like, leave the console running for several hours to monitor some program while i'm doing other work, i might be surprised if tasks suddenly start disappearing.

we could either add another flag to disable expiring completed completely (which would conflict with this flag), or we could accept `--retain-for=none` or something? i'm not sure which would be a better user experience for the CLI...

Reply:
Yes, I left that by mistake. Was testing it with 5 hours to ensure tasks are not cleared. having a `--retain-for=none` works for me